Britannia/localization/english/BTWK_cultural_names_l_english.yml

```diff
@@ -4737,7 +4801,7 @@
  cn_lochwinnoch_anglosaxon:0 "Lohwinnoh"
  
  cn_vanduensia:0 "Vanduensia"
- cn_vanduara_greek:0 "Ouandoúara"
+ cn_vanduara_greek:0 "Ouindogara"
```

Comment:
Should be consistent with the Latin form - either have Ouindogara/Vindogara, or the more conventional Ouandouara/Vanduara